src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Return Ball with number:" + number + " and " + color + " color";

```

Comment:
Typographical error: The string in the `toString` method should be more consistent with typical string representations. Consider changing "Return Ball with number:" to "Ball with number: ".